### PR TITLE
fixes ArrayIndexOutOfBoundsException when reading CSV file #658

### PR DESCRIPTION
**Pull request by lwhite1**
trims headers consistently.

Thanks for contributing.

- [x] Tick to sign-off your agreement to the [Developer Certificate of Origin (DCO) 1.1](https://developercertificate.org)

## Description

Trim headers consistently

## Testing

Did you add a unit test? Yes

### CODE REVIEW

**Comment by benmccann** [comment]
@lwhite1 Travis is failing

File: core/src/main/java/tech/tablesaw/io/FileReader.java [hunk 1]
```diff
@@ -62,6 +62,7 @@ private String cleanName(String name) {
       String[] headerNames = parser.parseNext();
       // work around issue where Univocity returns null if a column has no header.
       for (int i = 0; i < headerNames.length; i++) {
+        headerNames[i] = headerNames[i].trim();
```
Comment (benmccann) [inline core/src/main/java/tech/tablesaw/io/FileReader.java]: It looks like this could cause an NPE since the next line checks if `headerNames[i] == null`

Comment (lwhite1) [reply core/src/main/java/tech/tablesaw/io/FileReader.java]: i don't think it's possible

> @lwhite1 Travis is failing

It's the log-length thing again, which I've "fixed" several times by suppressing various kinds of output. I guess we're limited to somewhere 3000-3500 lines of output now or some such. This time the extra output is from issues related to documentation. 

With the sub-projects it seems we will run into this a lot. I'd like to suppress all logging except errors. Is that possible?

Comment (benmccann) [reply core/src/main/java/tech/tablesaw/io/FileReader.java]: I just pushed a fix

**Pull request merged**